bark_ml/library_wrappers/lib_tf_agents/tfa_wrapper.py

```diff
@@ -7,7 +7,9 @@
 from tf_agents.trajectories import time_step as ts
 
 tf.compat.v1.enable_v2_behavior()
-
+config = tf.compat.v1.ConfigProto()
+config.gpu_options.allow_growth = True
```

Comment:
This prevents that Tensorflow allocates 100% of the GPU memory. With this change multiple TFA agents can be trained simultaneously.